src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -182,6 +182,19 @@ public void forceStop() {
 		_phase = new Finished();
 		disableGame();
 	}
+
+	/**
+	* @return      0 if successful<br> 
+	* 				-1 if not in finished phase
+	*/
+	public int stop() {
+		if (!_phase.hasGameStarted() && _phase.isClockStopped()) {
```

Comment:
Maybe it is better to create a new method in phase class named something like hasBeenFisnished or something like that. Playing arround with already existent booleans makes it hard to understand what is going on.